src/requirements/CamelJBang.ts

```diff
@@ -16,22 +16,21 @@
  */
 'use strict'
 
-import { ShellExecution } from "vscode";
+import { ShellExecution, workspace } from "vscode";
 
 /**
  * Camel JBang class which allows shell execution of jbang cli init command
  */
 export class CamelJBang {
 
-	public static DEFAULT_CAMEL_VERSION = '3.20.4';
-
 	private camelVersion: string;
 
-	constructor(version?: string) {
-		this.camelVersion = version ? version : CamelJBang.DEFAULT_CAMEL_VERSION;
+	constructor() {
+		const camelJBangSettings = workspace.getConfiguration().inspect('camel.languageSupport.JBangVersion');
+		this.camelVersion = (camelJBangSettings.globalValue ? camelJBangSettings.globalValue : camelJBangSettings.defaultValue) as string;
```

Comment:
I'm surprised that we need to use the inspect. i would expect to use workspace.getConfiguration() directly.

With current code, I'm not sure that it is covering all cases of level to define settings: workspace, user, global. have you tried all of them?

Reply:
okey you are right.. I had to missed something when I was testing capabilities of workspace.getConfiguration API because it was not working as expected but maybe I messed up config ID I was looking for.

 It can be simplified to one line `this.camelVersion = workspace.getConfiguration().get('camel.languageSupport.JBangVersion') as string;`

thank you